data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,571 @@
+         .. new_objectives(obj_tribute)
+   }
+}
+
+tribute_not_started = {
+   {
+      title = _("Urgent Tribute Demand"),
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we haven’t received any help yet. If you are really on our side, you should start sending some wares urgently.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Kalitath
+         _([[As we are facing great losses due to your laziness, you shall now send us twice the previous amount of each ware.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Kalitath
+         _([[I am sure you do not want to break our alliance… do you?]]))
+   },
+   {
+      title = _("Blackmail!"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm us all. Our so-called ally is blackmailing us, and we can’t punish him for his insult as he has our princess at his hands.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Sidolus
+         _([[Our only chance is to double our efforts and start delivering something soon.]]))
+         .. new_objectives(obj_tribute2)
+   },
+}
+
+tribute_started = {
+   {
+      title = _("Tribute Started"),
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we have received the first batch of wares. So we may finally trust you.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Kalitath
+         _([[However, our enemy in the north is very strong. I am afraid that without significant help from your side we might lose the war despite any help in terms of wares you might provide.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Kalitath
+         _([[They name themselves an Empire, and their leader is called Maletus. I want you to train your soldiers and help us defeat his lousy Empire.]]))
+         .. new_objectives(obj_defeat_maletus)
+   },
+   {
+      title = _("Be Vigilant!"),
+      body = jundlina(_("Jundlina"),
+         -- TRANSLATORS: Jundlina
+         _([[May Satul warm you, Sidolus. Our ally now has gained some trust, but we shouldn't trust to much in them.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Jundlina
+         _([[So I want you to defeat this Empire, but ensure that we will become and remain strong enough to enforce the respect we deserve.]]))
+   },
+   {
+      title = _("Keep Delivering"),
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, I almost forgot to mention, but it should have been clear anyway.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Kalitath
+         _([[We are in urgent need of your supply, so please hurry up with your deliveries.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Kalitath
+         _([[If you fail to deliver everything within 3 hours, we need to assume you are just playing along with us.]]))
```

Comment:
```suggestion
         _([[If you fail to deliver everything within three hours, we need to assume you are just playing along with us.]]))
```

---

I am not sure if "playing along with us" is correctly phrased or not, someone else should comment on that…

Reply:
I lokked up "hinhalten" in German